components/itemDetailLayout/productdetaillayout.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import productList from "../../data/data";
+import * as detail from "./itemdetail.module.css";
+import Carousel from "../productImage/productImage";
+
+const productLayout = ()=> {
+	return (
+		<div className={detail.content}>
+			<div >
+				<Carousel/>
+				{/* <img src={productList[0].image} alt={productList[0].name}  /> */}
+			</div>
+
+			<div className={detail.text}>
+				<div className="p-1 space-y-5">
+					<h1 className="text-gray-700 text-center text-3xl font-bold text-md">
+						{productList[0].name}
+						<hr></hr>
+					</h1>
+					<p className="text-sm text-gray-900 text-left leading-sm">
+						Rate : 4
+					</p>
+					<p className="text-lg text-gray-900 text-left leading-sm">
+						{productList[0].description}
```

Comment:
Copy paste the description right now don't use product list . I am going to move it to different folder